### PR TITLE
A few tweaks to profile tutorial

### PR DESCRIPTION
**Pull request by jfy133**
# nf-core/eager pull request

Many thanks for contributing to nf-core/eager!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If necessary, also make a PR on the [nf-core/eager branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/eager)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --paired_end`).
- [x] Make sure your code lints ([`nf-core lint .`](https://nf-co.re/tools)).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by TCLamnidis** [comment]
Once we manage to get nuclear contamination onto MultiQC properly, I will also add the nuclear contamination column descriptions to `output.md`

File: docs/output.md [hunk 1]
```diff
@@ -110,6 +110,11 @@ The possible columns displayed by default are as follows:
 - **>= 1X** to **>= 5X** These are from Qualimap. This is the percentage of the genome covered at that particular depth coverage.
 - **% GC** This is the mean GC content in percent of all mapped reads post-deduplication. This should normally be close to the GC content of your reference genome.
 - **MT to Nuclear Ratio** This from MTtoNucRatio. This reports the number of reads aligned to a mitochondrial entry in your reference FASTA to all other entries. This will typically be high but will vary depending on tissue type.
+- **XRate** This is from Sex.DetERRmine. This is the relative coverage on the X-chromosome.
```
Comment (jfy133) [inline docs/output.md:113]:
```suggestion
- **XRate** This is from Sex.DetERRmine. This is the relative depth coverage on the X-chromosome.
```

File: docs/output.md [hunk 2]
```diff
@@ -110,6 +110,11 @@ The possible columns displayed by default are as follows:
...
+- **YRate** This is from Sex.DetERRmine. This is the relative coverage on the Y-chromosome.
```
Comment (jfy133) [inline docs/output.md:114]:
```suggestion
- **YRate** This is from Sex.DetERRmine. This is the relative depth coverage on the Y-chromosome.
```

File: docs/output.md [hunk 3]
```diff
@@ -110,6 +110,11 @@ The possible columns displayed by default are as follows:
...
+- **Number of SNPs** This is from ANGSD. The number of SNPs left after removing sites with no data in a 5 base pair surrounding region.
+- **Contamination Estimate (Method1_ML)** This is from ANGSD. The Maximum Likelihood contamination estimate according to Method 1. The estimates using Method of Moments and/or those based on Method 2 can be unhidden through the "Configure Columns" button.
```
Comment (jfy133) [inline docs/output.md:116]:
```suggestion
- **Contamination Estimate (Method1_ML)** This is from the nuclear contamination function of ANGSD. The Maximum Likelihood contamination estimate according to Method 1. The estimates using Method of Moments and/or those based on Method 2 can be unhidden through the "Configure Columns" button.
```

Is there a simple way of describing what a user is looking for (i.e. high or low value) @TCLamnidis ?

Comment (TCLamnidis) [reply docs/output.md:116]: That would hinge on their specific cutoffs. Most people use 5% (`0.05`) to my knowledge, but some do change that cutoff.

**Comment by TCLamnidis** [comment]
> @TCLamnidis some comments but otherwise good to go. You'll need to fix and then also approve so I can merge (I can't approve my own PR ;))

I should fix the MQC sample name first.

File: docs/tutorial_profiles.md [hunk 4]
```diff
@@ -138,7 +144,12 @@ command:
 nextflow run nf-core/eager -c old_dna_profile.config -profile old_dna,hpc_blue <...>
 ```
 
-Then the final profile used by your given run would look like:
+In the background, any parameters in the pipeline's `nextflow.config` (containing
+default parameters) will be overwritten by the `old_dna_profile.config`, but
+in addition, *profile* will overwrite any parameters set in the config but
+outside the profile.
```
Comment (TCLamnidis) [inline docs/tutorial_profiles.md:150]:
```suggestion
default parameters) will be overwritten by the `old_dna_profile.config`. In addition, the `old_dna` *profile* will overwrite any parameters set in the config but
outside the profile definition of `old_dna_profile.config`.
```

**Comment by jfy133** [comment]
> docs/images/tutorials/profiles/config_profile_inheritence.png/svg is confusing (as is the material it explains ofc). Would be good to add an explanation of what the arrows mean on the figure. Currently they can be interpreted as "overwrites", "is contained/defined in", or an order of operations.
> 
> Also "here assumes 'silent' loading" is a broken sentence. Who assumes? why not just say "Nextflow will silently load a second config file that includes the definition of profile b and is loaded from the nf-core repository" or sth? Also highlight which profile is silently loaded. maybe rename it to (`silent.conf`) or something. it looks the same as the default config atm.
> 
> Additionally, without further explanation it seems like nextflow is magic and knows where profile b is. I think you should specifically mention that it is loaded remotely from specific local or remote addresses.

Fixed in https://github.com/nf-core/eager/pull/507/commits/2ce415e58f71356cffc606a148db513cff17a320

**Pull request merged**